x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java

```diff
@@ -382,6 +382,12 @@ private static InternalAggregation unrollMultiBucket(InternalMultiBucketAggregat
             });
         } else if (rolled instanceof StringTerms) {
             return unrollMultiBucket(rolled, original, currentTree, (bucket, bucketCount, subAggs) -> {
+
+                // Hide our `null_value` placeholder so it doesn't show up in the terms list
+                if (bucket.getKeyAsString().equals(Rollup.ROLLUP_NULL_VALUE)) {
+                    return null;
+                }
```

Comment:
Why is this not needed for LongTerms and the histograms? do they not have the concept of a null bucket as well?